app/helpers/buyers/applications_helper.rb

```diff
@@ -6,6 +6,7 @@ def metadata_new_app(buyer, provider)
       data-services_contracted='#{ services_contracted(buyer) }'
       data-service_plan_contracted_for_service='#{ service_plan_contracted_for_service(buyer) }'
       data-relation_service_and_service_plans='#{ relation_service_and_service_plans(provider) }'
+      data-create_service_plan_path='#{ create_service_plan_path }'
```

Comment:
I haven't tried but this might work just with a proc here instead of creating a method that is not useful for anything else. So it would be:
`data-create_service_plan_path='#{ proc { admin_service_service_plans_path ':service_id' } }'`